src/KOKKOS/mliap_descriptor_so3_kokkos.cpp

```diff
@@ -0,0 +1,278 @@
+/* ----------------------------------------------------------------------
+   LAMMPS - Large-scale Atomic/Molecular Massively Parallel Simulator
+   https://www.lammps.org/, Sandia National Laboratories
+   Steve Plimpton, sjplimp@sandia.gov
+
+   Copyright (2003) Sandia Corporation.  Under the terms of Contract
+   DE-AC04-94AL85000 with Sandia Corporation, the U.S. Government retains
+   certain rights in this software.  This software is distributed under
+   the GNU General Public License.
+
+   See the README file in the top-level LAMMPS directory.
+------------------------------------------------------------------------- */
+
+/* ----------------------------------------------------------------------
+   Contributing author: Matt Bettencourt (NVIDIA)
+ ------------------------------------------------------------------------- */
+
+#include "mliap_descriptor_so3_kokkos.h"
+
+#include "atom_kokkos.h"
+#include "comm.h"
+#include "error.h"
+#include "memory.h"
+#include "mliap_data_kokkos.h"
+#include "mliap_so3_kokkos.h"
+#include "pair_mliap.h"
+#include "tokenizer.h"
+
+#include <cstring>
+
+using namespace LAMMPS_NS;
+
+#define MAXLINE 1024
+#define MAXWORD 3
+
+/* ---------------------------------------------------------------------- */
+template <class DeviceType>
+MLIAPDescriptorSO3Kokkos<DeviceType>::MLIAPDescriptorSO3Kokkos(LAMMPS *lmp, char *paramfilename)
+ // TODO: why take self as param, shouldn't be needed
+  :  Pointers(lmp), MLIAPDescriptorSO3(lmp, paramfilename), MLIAPDescriptorKokkos<DeviceType>(lmp, this)
+{
+  // TODO: the MLIAP_SO3 object likely needs a kokkos-ified version
```

Comment:
Double check this comment, it looks outdated.

Reply:
was